ena-submission/src/ena_deposition/call_loculus.py

```diff
@@ -140,13 +145,13 @@ def fetch_released_entries(config: Config, organism: str) -> Iterator[dict[str,
 
```

Comment:
This was the substantive change I noticed – assuming it's intentional, lgtm

Reply:
Uh looks like this appeared I. The wrong place – the 3600 below I meant

Reply:
oh wupps - this should be merged - think I need to rebase!

Reply:
thanks for the thorough review!

Reply:
ah wupps it was only in the cherry pick: https://github.com/loculus-project/loculus/pull/3611